variety/plugins/builtin/downloaders/EarthviewDownloader.py

```diff
@@ -30,15 +30,19 @@
 
 class EarthviewDownloader(SimpleDownloader):
     DESCRIPTION = _("Google Earth View Wallpapers")
-    ROOT_URL = "https://earthview.withgoogle.com/"
+    ROOT_URL = "https://www.gstatic.com/prettyearth/assets/data/v3/"
+    DATA_URL = ROOT_URL + "{photo_id}.json"
+    # PHOTO_IDS fetched from JS source of Earth View Chrome extension:
+    # https://chromewebstore.google.com/detail/earth-view-from-google-ea/bhloflhklmhfpedakmangadcdofhnnoh?hl=en
```

Comment:
Is there any alternative method to get the valid IDs, it's pretty ugly having them hardcoded and eventually people will start asking "why is this never showing the newest images"? 
Also, please run the file through "black", this long line should be split - https://github.com/varietywalls/variety/blob/master/CONTRIBUTING.md#code-style.

Reply:
There is https://github.com/alexpersian/earthview, which is an up-to-date fork of the method from https://github.com/limhenry/earthview that was used before. 

Since both the (likely out-of-date) Firefox extension and the Chrome extension hardcode the IDs, I think it's likely that there is no longer an *official* hosted ID list.

Reply:
Ok, clear, thanks.

Reply:
So for this comment, let's just make sure we run black on the code. Thanks.

Reply:
Just to be clear, the Chrome extension is developed by Google (last updated July 3, 2024) and they also use a hardcoded IDs, so I think the best we can do is copy them :shrug:

Reply:
I ran black and I dont love it because it adds ~2600 lines of code to the file.  Lines that aren't _really_ code and dont _really_ need to be human readable IMHO.

Reply:
What if we just have a separate file `photoIds.txt` that contains all of the IDs, and we load from that?